packages/webdriverio/src/commands/mobile/swipe.ts

```diff
@@ -0,0 +1,235 @@
+import logger from '@wdio/logger'
+
+import type { SwipeOptions, XY } from '../../types.js'
+import { MobileScrollDirection } from '../../types.js'
+
+const log = logger('webdriverio')
+const SWIPE_DEFAULTS = {
+    DIRECTION: MobileScrollDirection.Down,
+    DURATION: 1500,
+    PERCENT: 0.95,
+}
+
+/**
+ *
+ * Swipe in a specific direction within viewport or element for Desktop/Mobile Web <strong>AND</strong> Mobile Native Apps.
+ *
+ * :::info
+ *
+ * Swiping for Mobile Native Apps is based on the W3C-actions protocol, simulating a finger press and movement.
+ * This is different from the [`mobile: scrollGesture`](https://github.com/appium/appium-uiautomator2-driver/blob/master/docs/android-mobile-gestures.md#mobile-scrollgesture) for Android
+ * or [`mobile: scroll`](https://appium.github.io/appium-xcuitest-driver/latest/reference/execute-methods/#mobile-scroll) for iOS command which is based on the Appium Driver protocol and is
+ * only available for mobile platforms in the NATIVE context.
+ *
+ * :::
+ *
+ * :::caution Swiping based on coordinates
+ *
+ * Avoid using `from` and `to` options unless absolutely necessary. These are device-specific and may not work consistently across devices.
+ * Use the `scrollableElement` option for reliable swipes within an element.
+ *
+ * :::
+ *
+ * <example>
+    :swipe.js
+    it('should execute a default swipe', async () => {
+        await browser.swipe();
```

Comment:
I am not sure what this command does without parameter 🤔 if the default swipe is from left to right, let's mention it here in the example

Reply:
Found I made an error, thanks for pointing this out. Will push a fix in a few min, but the default will be `up` :D